test-fuzz/tests/integration/ci.rs

```diff
@@ -232,7 +232,16 @@ fn unmaintained() {
         .success();
 }
 
+#[allow(deprecated)]
+#[test]
+fn actionlint() {
+    let home = env::home_dir().unwrap();
```

Comment:
```suggestion
    #[allow(deprecated)]
    let home = env::home_dir().unwrap();
```
Nit. Please move the `#[allow(deprecated)]` to here.